doc/install/mne_python.rst

```diff
@@ -186,9 +186,10 @@ Python development are:
   use Python. It is included by default in the ``base`` environment when you
   install Anaconda, and can be started from a terminal with the command
   ``spyder`` (or on Windows or macOS, launched from the Anaconda Navigator GUI).
-  If you use Spyder, you should *not* install it into the ``mne`` environment;
-  instead, launch Spyder from the ``base`` environment and set it to use the
-  ``mne`` environment automatically, by opening Spyder and navigating to
+  It can also be installed with `dedicated installers <https://www.spyder-ide.org/>`_.
```

Comment:
The link does not point to the installers. But I'm not sure if we want to inform about the standalone Spyder installers if we've not tested if they fit the procedure outlined in these instructions.

Reply:
Installers should work with the procedure in this PR (2 venv). If you find a better link, feel free to swap, but there is no link pointing directly to the Download section, and the Download button link is OS specific..

Reply:
nevermind, `https://www.spyder-ide.org/#section-download`